### ISSUE TITLE
Dependency update breaks service mapping

### ISSUE THREAD

**Issue by lukeheath** [Ignigena/nautilus#1]
Hi @Ignigena! We ran into an issue today with Nautilus and I wanted to give you a heads up. 

On Friday, [require-all](https://www.npmjs.com/package/require-all) published an update that broke Nautilus mapping of services from the file path.

When we attempted to re-build a previously successful build, we received this error: 

```
ReferenceError: ContentfulService is not defined
    at Object.module.exports [as contentful] (/Users/LukeH/projects/web/hooks/contentful.js:9:3)
    at _.each.hook (/Users/LukeH/projects/nautilus/index.js:70:21)
    at arrayEach (/Users/LukeH/projects/nautilus/node_modules/lodash/lodash.js:537:11)
    at Function.forEach (/Users/LukeH/projects/nautilus/node_modules/lodash/lodash.js:9344:14)
    at Nautilus.loadHooks (/Users/LukeH/projects/nautilus/index.js:63:7)
    at new Nautilus (/Users/LukeH/projects/nautilus/index.js:40:10)
    at Object.<anonymous> (/Users/LukeH/projects/web/index.js:8:14)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

We went through Nautilus dependencies and saw that `require-all` had just published `2.1.0`. We locked the version down to `2.0.0` and service mapping was restored.

**Comment by Ignigena** [comment]
Thanks for the heads up! I'll look into the change that was introduced.

Do you need an upstream fix or did locking down the version suffice for you? Reason I ask is because the next release is planned to be breaking (don't worry, will semver responsibly!) specifically due to the deprecation of superglobal evils.

**Comment by lukeheath** [comment]
Sure thing! We just forked and locked down the dependency for the time being, and that fixed it.

**Comment by Ignigena** [comment]
The issue should be addressed in 8b3bd57031700232d1d43471e95c6ceefcd17eb6 although I likely won't be rolling version 0.6 until end of next week. The biggest breaking change for y'all should just be the deprecation of superglobals. Instead of `api/location/location.service.js` producing a `LocationService` global you'll need to use the `app.api.service.location` address.

Everything else is pretty much centered around API scaffolding and splitting the project into Nautilus core and web so some of the core functionality could be re-used in an Electron flavor of Nautilus.

Hit me up if you have any questions. Leaving the issue open in the meantime to make sure it was addressed properly :)

**Comment by lukeheath** [comment]
Cool thanks for the update! We should be good to go so going ahead and closing the issue. Hope all is well! :)
